packages/dx-react-scheduler/src/plugins/drag-drop-provider.tsx

```diff
@@ -311,22 +313,31 @@ class DragDropProviderBase extends React.PureComponent<
 
         <Template name="allDayPanel">
           <TemplatePlaceholder />
-          <TemplateConnector>
-            {({ formatDate, resources, plainResources }) => renderAppointmentItems(
-              this.allDayDraftAppointments, formatDate, draftData,
-              Container, DraftAppointment, resources, plainResources,
-            )}
-          </TemplateConnector>
+          <TemplatePlaceholder
+            name="draftAppointment"
+            params={{ data: draftData, draftAppointments: this.allDayDraftAppointments }}
+          />
```

Comment:
This template names `draftAppointment` but renders an array of drafts.

I think we should render an array of `TemplatePlaceholders` here.